packages/forms-web-app/src/controllers/common/enter-code.js

```diff
@@ -319,6 +323,10 @@ const postEnterCodeLPA = (views) => {
 		if (!tokenVerification(res, token, views, id)) return;
 
 		try {
+			const currentUserStatus = getLPAUserStatus(id);
```

Comment:
think there's a missing await here

Reply:
Good spot. Fixed unit test to mockResolvedValue also